### DISCUSSION TITLE
Simulating Twisted Bilayer Graphene (tBLG) with GPUMD: Issue with Combining Potentials

### DISCUSSION THREAD

**Discussion by akabli-younes** [brucefan1983/GPUMD#914] [General]
Dear Developers,

I am trying to simulate twisted bilayer graphene (tBLG) using the latest version of GPUMD, aiming to reproduce results from existing studies and to compare them with the results I have already obtained using LAMMPS.

However, I am facing difficulties when using a combination of LJ (Lennard-Jones) or ILP (interlayer potential) with the Tersoff potential. While the simulation works well when using only the Tersoff potential, issues arise when I try to combine it with either LJ or ILP.

the potential files for reference. 
[C_15.txt](https://github.com/user-attachments/files/17774295/C_15.txt)
[ilp_tmd_sw_ILP_HBNC_oywg_2018.txt](https://github.com/user-attachments/files/17774296/ilp_tmd_sw_ILP_HBNC_oywg_2018.txt)
Does anyone have experience with using these potentials together in GPUMD or advice on resolving this issue? Any suggestions or insights would be greatly appreciated!

**Comment by BBBuZHIDAO** [comment]
Hi, it is so sorry that the ILP in the newest version is just for TMD system. Actually the potentials are ILP and SW. So now this ILP could not simulate graphene. :cry: 
But you can use the old version where LJ could combine the Tersoff for tBLG. (v3.3.1)

**Comment by BBBuZHIDAO** [comment]
Hello, these days GPUMD has updated a new hybrid potential **NEP+ILP** for graphene and h-BN systems. It could be used to simulate tBLG, too. The usage is in #835 and will update in official doc later. May this method could help you! If any other questions, please feel free to let me know!
